mandatory/1-fix-functions.js

```diff
@@ -9,10 +9,8 @@
   NOTE:Only make edits inside the function
 
 */
-
-function mood() {
-  let isHappy = true;
-
+let isHappy = true;
```

Comment:
no need to add isHappy variable here we take isHappy from parameter when we call this function we gave true or false as an argument, under the tests see they already given arguments to this function